backend/src/main/java/com/empire/Character.java

```diff
@@ -81,16 +89,20 @@ public void addExperienceGovernor() {
 		experience.governor += Constants.oneDimExpAdd;
 	}
 
-	public boolean hasTag(String tag) {
+	public boolean hasTag(Tag tag) {
 		return tags.contains(tag);
 	}
 
-	void addTag(String tag) {
+	void addTag(Tag tag) {
 		tags.add(tag);
 	}
 
-	void removeTag(String tag) {
+	void removeTag(Tag tag) {
 		tags.remove(tag);
 	}
+
+	boolean captive() {
```

Comment:
One final thing, I think this should be made to isCaptive() so the resulting code reads more like a sentence.

Reply:
Done.